frontend/src/components/TopBar.tsx

```diff
@@ -43,25 +40,27 @@ function regions(selected: any) {
       ['Europe (Frankfurt)', 'eu-central-1'],
       ['Europe (Ireland)', 'eu-west-1'],
       ['Europe (London)', 'eu-west-2'],
-      ['Europe (Milan)', 'eu-south-1'],
       ['Europe (Paris)', 'eu-west-3'],
       ['Europe (Stockholm)', 'eu-north-1'],
     ],
-    [['Middle East (Bahrain)', 'me-south-1']],
     [['South America (São Paulo)', 'sa-east-1']],
-    [
-      ['China (Beijing)', 'cn-north-1'],
-      ['China (Ningxia)', 'cn-northwest-1'],
-    ],
   ]
 
-  if (selected === 'us-gov-west-1')
-    supportedRegions = [
-      [
-        //["US Gov (east-1)", "us-gov-east-1"],
-        ['US Gov (west-1)', 'us-gov-west-1'],
-      ],
-    ]
+  const optInRegions = [
+    ['Africa (Cape Town)', 'af-south-1'],
+    ['Asia Pacific (Hong Kong)', 'ap-east-1'],
+    ['Europe (Milan)', 'eu-south-1'],
+    ['Middle East (Bahrain)', 'me-south-1'],
+    ['China (Beijing)', 'cn-north-1'],
+    ['China (Ningxia)', 'cn-northwest-1'],
+    ['AWS GovCloud (US-East)', 'us-gov-east-1'],
+    ['AWS GovCloud (US-West)', 'us-gov-west-1'],
+  ]
+
+  const optInRegion = optInRegions.filter(region => region[1] === selected)
```

Comment:
question: this seems to show only the opt-in region in which PCM has been installed. What if I have it installed in another region but I want to create clusters in an opt-in one? Shouldn't we support all the regions a customer has opted-in rather than just the one he might have installed PCM?

Reply:
You are correct, support for other opt-in regions is out of the scope of this PR (more details in the related task tracker)